src/geo/cartodb-layer-group.js

```diff
@@ -142,6 +142,7 @@ var CartoDBLayerGroup = Backbone.Model.extend({
     if (staticImageURLTemplate) {
       staticImageURLTemplate = this._appendParamsToURL(staticImageURLTemplate, [ 'layer=' + this._getIndexesOfVisibleLayers() ]);
       staticImageURLTemplate = this._appendAuthParamsToURL(staticImageURLTemplate);
+      staticImageURLTemplate = staticImageURLTemplate.replace('{s}', this.getSubdomains()[0]);
```

Comment:
Yes. You're right, but... 

1. Since this link is probably going to be used a single time I figured we could just use the first available subdomain.

2. `getSubdomains` can return an empty array, but in that case the template won't have a `{s}` so the replace will have no effect (and it'll work fine with `undefined`).

If you have a better suggestion... I'm all 👂